code-of-conduct.md

```diff
@@ -12,21 +12,33 @@ We expect everyone involved in our community to follow this code of conduct. Thi
 
 We do not tolerate harassment of people at our events or space in any form. 
 
+
+We do not seek to list all cases of unacceptable behavior, but provide examples to help guide our community in thinking through how to respond when we experience these types of behavior, whether directed at ourselves or others. 
+
+Some guidelines on appropriate behavior:
+
+* If you are unsure if something is appropriate behavior, err on the side of caution. 
+* Each person you interact with can define what is appropriate for them. 
+* Impact matters more than intent. 
+* It is your responsibility to ensure that your behavior does not have a negative impact.
+* Problems happen when we assume that our way of thinking or behaving is the norm or ok with everyone. This is particularly problematic when we are in a position of power or privilege.
```

Comment:
I think these lines are what I have the most issue with out of these proposed changes. They feel like they're written with good intent but haven't been battle tested with really crappy people who would abuse them.

Things like "impact matters more than intent" sound great, but the reality is they both matter and there's lots of situations where you do need to understand intent as a lens for the impact. If someone gets hit in the head with a board, the first questions will be all about context, who was paying attention, whether someone was being negligent, etc. Similarly, "each person you interact with can define what is appropriate for them" can be abused pretty easily. "Hey I think rainbows are disrespectful to Christians, so I don't like that you're displaying rainbows" for example.

I'm wondering if it wouldn't be better to focus on things that people might not see as problematic, but end up being problematic. For example, if you're a woman in the space and every other guy comes up and offers you advice, but they wouldn't do that for other men in the space, that's not harassment per se, but it is objectively annoying and it's a common thing people don't think about.

Reply:
I think this is a great point, thank you for sharing it. 

I am wondering if the concern about bad actors can be addressed by rewording these points, or if what we need is a different framework/complete rewrite of this section. I'm fine with either way to move forward and might experiment with both, but value your input if you have specific thoughts on it. I am also wondering if this should be added to the code of conduct directly or if it should be a separate document with potentially more examples and an anti-harassment-training tone.

Reply:
This is the only section I think needs rewrites. 

There's a reason a lot of corporate CoCs leave out guidelines like this, and that's because a lot of the guidelines end up being interpreted as gospel and used as rules.

Saying something like "denhac's membership is diverse and comes from different backgrounds, histories and experiences. Problems may occur when we assume that everyone shares our viewpoints or ways of thinking. Careless words and actions can affect other people in unintended ways. We expect all members to treat other members respectfully to promote a positive atmosphere within the space."

This doesn't tell people exactly how to treat other people but instead warns people of the dangers of not doing the things in the bullet points above while not explicitly giving power to distinct actions?

I think we can re-write this to not provide a bulleted list of